modules/lvms-scaling-storage-expand-pvc.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-scaling-expand-pvc_{context}"]
+= Expanding PVCs
+
+To leverage the new storage after adding additional capacity, you can expand existing persistent volume claims (PVCs) with LVM Storage.
```

Comment:
My ignorance here. Is there a specific way folks need to add capacity in order to leverage this feature? (Likely in a previous section??). Do we need to explicitly state any thing? It seems vague to me. (Again, I know little of storage.)